pack/rpm.mk

```diff
@@ -81,6 +81,7 @@ $(BUILDDIR)/$(RPMSPEC): $(RPMSPECIN)
                 -re 's/(%autosetup.*)( -n \S*)(.*)/\1\3/' \
 		-e '0,/%autosetup.*/ s/%autosetup.*/%autosetup -n $(PRODUCT)-$(VERSION)/' \
                 -e '/%changelog/a\* $(THEDATE) $(CHANGELOG_NAME) <$(CHANGELOG_EMAIL)> - $(VERSION)-$(RELEASE)\n\- $(CHANGELOG_TEXT)\n' \
+		-e 's/IS_TRAVIS/$(IS_TRAVIS)/' \
```

Comment:
I would use defines instead of patching of the spec where possible. Let's add them to both stages: build of source RPM and a package itself.

I also propose to name the variable `TRAVIS_CI` and name define `_travis_ci`, because IS_TRAVIS looks strange (should be at least IS_IN_TRAVIS) and to unify with `GITLAB_CI` (see below).

I also propose to pass `CI` and `GITLAB_CI` variables in the same way (to defines `_ci` and `_gitlab_ci`).

I think we can add the same variables to debuild's `--preserve-envvar` option: `CI`, `TRAVIS_CI`, `GITLAB_CI`.

Don't sure whether we should change TRAVIS to TRAVIS_CI (_travis_ci define) or keep it be TRAVIS (_travis).

Let's describe passed environment variables and defines in README.